src/blockjacobi.jl

```diff
@@ -0,0 +1,60 @@
+using SuiteSparse
+using SparseArrays
+using LinearAlgebra
+mutable struct BlockJacobi{T, S<:AbstractMatrix{T}} <: LSPreconditioners.Preconditioner
+    nblocks :: Int64
+    blocksizes :: Array{Int32}
+    blocks :: Vector{Union{LU{T, Matrix{T},Vector{Int32}},SuiteSparse.UMFPACK.UmfpackLU{Float64,Int64}}}
```

Comment:
@npritch928 You use inconsistent spacing all through the code. For instance

`LU{T, Matrix{T},Vector{Int32}`

You have space after T but not after Matrix{T}. You have to have a consistent style all through the code. Please, refer either to style guides from Julia

https://docs.julialang.org/en/v1/manual/style-guide/
https://github.com/invenia/BlueStyle

Or code in other libraries.